tests/integration/test_install_uri.py

```diff
@@ -143,9 +143,6 @@ def test_install_named_index_alias(pipenv_instance_private_pypi):
 @pytest.mark.index
 @pytest.mark.install
 @pytest.mark.needs_internet
-@pytest.mark.skipif(
-    sys.version_info >= (3, 12), reason="Package does not work with Python 3.12"
-)
```

Comment:
This test was failing at one point, and I discovered that it could pass on 3.13 -- I think we had a different bug at one point that lead to this.